addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/SqueezeBoxHandlerFactory.java

```diff
@@ -52,6 +59,9 @@
     private Map<ThingUID, ServiceRegistration<?>> discoveryServiceRegs = new HashMap<>();
 
     private AudioHTTPServer audioHTTPServer;
+    private NetworkAddressService networkAddressService;
+
+    private String callbackUrl;
```

Comment:
nevermind, I have removed the member variable and just calculate the url on the fly and do not store it in the class anymore.